src/components/Form.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const Form = () => (
+  <div className="container py-3">
+    <hr style={{ height: '3px' }} />
```

Comment:
- [ ] Let's try to avoid inline styling as it is not best practice. Kindly move all styles to an external CSS file.